xds/type/matcher/v3/cel.proto

```diff
@@ -30,8 +27,7 @@ option (xds.annotations.v3.file_status).work_in_progress = true;
 // Refer to :ref:`Unified Matcher API <envoy_v3_api_msg_.xds.type.matcher.v3.Matcher>` documentation
 // for usage details.
 //
-// [#comment:TODO(sergiitk): Link HttpAttributesMatchInput + usage example.]
```

Comment:
@sergiitk is removing the TODOs make sense to you, or should more examples be added?

Reply:
Yeah, i have thought about this.  I feel examples is probably overkill here. Looking through files in xds/type/matcher/v3, I only see string.proto has simple examples.

But I can keep this TODO if @sergiitk want.

Reply:
In this todo, I've probably meant `HttpAttributesCelMatchInput`, not `HttpAttributesMatchInput`.  Regarding what to link, I found this: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/matching/matching_api.html#custom-matching-algorithms. I think it'll fit perfectly here.

Reply:
Unified Matcher API is already linked  in the comment above. Also, `envoy.matching.matchers.cel_matcher` should provides enough information for customer to navigate.

Reply:
Up to you - but the Unified Matcher API link above points to the generated proto page (https://www.envoyproxy.io/docs/envoy/latest/xds/type/matcher/v3/matcher.proto#envoy-v3-api-msg-xds-type-matcher-v3-matcher), while https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/matching/matching_api.html#custom-matching-algorithms is more of a user-friendly page that explains how to use the Matcher API. 

I'm fine with the doc as it is though.